modules/plotting/include/modules/plotting/properties/plottextproperty.h

```diff
@@ -61,6 +61,7 @@ class IVW_MODULE_PLOTTING_API PlotTextProperty : public BoolCompositeProperty {
     FloatVec4Property color_;
     FloatProperty position_;  //!< position along axis [0,1]
     FloatProperty offset_;    //!< offset from axis
+    FloatProperty rotation_;  //!< Degrees of rotation
```

Comment:
consider `// rotation [degrees]` or `// rotation in degrees`
maybe mention that 0 degree means horizontal/regular text direction